api/v3/Contribution.php

```diff
@@ -99,7 +99,6 @@ function _civicrm_api3_contribution_create_spec(&$params) {
   $params['contact_id']['api.required'] = 1;
   $params['total_amount']['api.required'] = 1;
   $params['payment_instrument_id']['api.aliases'] = array('payment_instrument');
-  $params['receive_date']['api.default'] = 'now';
```

Comment:
Receive date can be null when status is <> 'Completed', 'Partially paid', 'Pending refund'